src/app.rs

```diff
@@ -1012,6 +1012,14 @@ impl App {
                 'j' | 'k' | 'g' | 'G' => self.handle_char(caught_char),
                 _ => {}
             }
+        } else if self.delete_dialog_state.is_showing_dd {
+            match caught_char {
+                'h' => self.on_left_key(),
+                'j' => self.on_left_key(),
+                'k' => self.on_right_key(),
+                'l' => self.on_right_key(),
```

Comment:
Very minor nit, since two of the branches are sharing the same outcome, I would personally do:

```rust
match caught_char {
    'h' | 'j' => self.on_left_key(),
    'k' | 'l' => self.on_right_key(),
}
```

But LGTM.

Reply:
Oh, thanks, makes sense.